src/main/java/com/kodilla/ecommercee/CartController/CartController.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.ecommercee.CartController;
+
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartDto;
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartItemDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("v1/carts")
+@RequiredArgsConstructor
+public class CartController {
+
+    @GetMapping("/cart")
+    public List<CartItemDto> getCart() {
+        CartItemDto cartItem1 = new CartItemDto(1L, "Kurtka", "Kurtka elegancka jesienna", 750, 2);
+        CartItemDto cartItem2 = new CartItemDto(2L, "Kurtka", "Kurtka sportowa jesienna", 850, 2);
+        CartItemDto cartItem3 = new CartItemDto(3L, "Kurtka", "Kurtka zimowa narciarska", 1200, 5);
+        List<CartItemDto> cart = new ArrayList<>();
+        cart.add(cartItem1);
+        cart.add(cartItem2);
+        cart.add(cartItem3);
+
+        return cart;
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> addCartItem(@RequestBody CartItemDto cartItemDto) {
+        return ResponseEntity.ok().build();
+    }
+
+    @DeleteMapping(value = {"cartItemId"})
+    public ResponseEntity<Void> removeCartItem(@PathVariable Long cartItemId) {
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/order")
+    public ResponseEntity<CartDto> createOrder() {
+        //I don't know what to put in here. Need ideas ;)//
+        return ResponseEntity.ok().build();
+    }
```

Comment:
You still don't have:
- add product to the cart
- delete product from the cart

@PutMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
public ResponseEntity<Void> addProductToCart(@RequestParam long cartId, @RequestParam long productId) {
     return ResponseEntity.ok().build();
    }

@PutMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
public ResponseEntity<Void> removeProductFromCart(@RequestParam long cartId, @RequestParam long productId) {
     return ResponseEntity.ok().build();
    }

Reply:
Please refer to commit:
[JDP230102-3_CartController_initial_update_1](https://github.com/interceptor2048/project-jdp-2301-02/pull/8/commits/53a0e8698bcec7cf6306df4f688ec9e5b25f1c34)

    @PostMapping(value = "/addItem")
    public ResponseEntity<Void> addCartItem(@RequestBody CartItemDto cartItemDto) {
        //post item into Cart//
        return ResponseEntity.ok().build();
    }

    @DeleteMapping(value = "/{cartItemId}")
    public ResponseEntity<Void> removeCartItem(@PathVariable Long cartItemId) {
        return ResponseEntity.ok().build();
    }